test.js

```diff
@@ -19,7 +19,9 @@ describe('ncs', function(){
 		assert.equal(ncs.rgb(''), null);
 		assert.equal(ncs.hex('NCS5000-N'), null);
 		assert.equal(ncs.hex('12345'), null);
+		assert.equal(ncs.hex('NCS 5000-Z20Y'), null);

```

Comment:
Out of the scope of this PR, but could you make it so that all the `null` tests use `strictEqual`? That would probably be handy to catch even more possible edge cases.